.gitignore

```diff
@@ -0,0 +1,5 @@
+﻿################################################################################
+# This .gitignore file was automatically created by Microsoft(R) Visual Studio.
+################################################################################
+
+/.vs
```

Comment:
Ik kan aanbevelen om de `.vs` folder (net als de `.idea` folder) in je global ignore van je pc te zetten :)